core/src/main/java/nl/nn/adapterframework/http/rest/ApiListenerServlet.java

```diff
@@ -463,7 +463,11 @@ else if(segment.startsWith("{") && segment.endsWith("}")) {
 					}
 					messageContext.put("multipartAttachments", attachments.toXML());
 				} else {
-					body = parseContentAsMessage(request.getInputStream(), request.getContentType());
+					if(request.getContentLength() > -1) { //If content is present (POST/PUT) this header must be set (see https://www.rfc-editor.org/rfc/rfc7230#section-3.3)
```

Comment:
Je moet de spec goed lezen. In plaats van contentLength >=0 kan ook Transfer-Encoding=chunked gebruikt worden